### ISSUE TITLE
Select monster on user view

### ISSUE THREAD

**Issue by tstrat** [tstrat/personal-project#2]
To be done AFTER #1 Adding monster.

Given any number of monsters on the board, user should be able to select any subset of those monsters to perform actions with.

*  Limit select to monsters of the same type (for ease of use in later sections)
*  Start with one selected
*  Should monsters be unhallowed to be selected if they have already done actions ?
    * Probably not since it adds more complexity to user experience
